pkg/run/run.go

```diff
@@ -61,10 +56,10 @@ func (b *DockerRunner) Run(config *api.Config) error {
 
 	go docker.StreamContainerIO(errReader, nil, glog.Error)
 	go docker.StreamContainerIO(outReader, nil, glog.Info)
-	rerr := b.ContainerClient.RunContainer(opts)
-	if e, ok := rerr.(errors.ContainerError); ok {
-		return errors.NewContainerError(config.Tag, e.ErrorCode, errOutput)
-	}
 
+	err := b.ContainerClient.RunContainer(opts)
+	if e, ok := err.(errors.ContainerError); ok {
+		return errors.NewContainerError(config.Tag, e.ErrorCode, "")
```

Comment:
(not a required answer to get this merged, but if it turns out to be not needed we can remove it here or in a follow up)